test/config_loader.test.js

```diff
@@ -505,6 +505,7 @@ function test_loadDefaults() {
   it("Loads from default.json file", async () => {
     let lc = new LoaderContext("test/dummy_configs");
     let result = await lc.loadConfigDefaults();
+    console.log(await readFile("./test/dummy_configs/default.json"));
```

Comment:
`readFile` actually returns a `Buffer` (it should return a `string`).

This `console.log` still shouldn't be here.